pkg/kubeapiserver/admission/config.go

```diff
@@ -67,7 +67,6 @@ func (c *Config) New(proxyTransport *http.Transport, egressSelector *egressselec
 	discoveryRESTMapper := restmapper.NewDeferredDiscoveryRESTMapper(discoveryClient)
```

Comment:
discoveryClient / discoveryRESTMapper are unused now, move these declarations and the `admissionPostStartHook` to reset / populate it to k8s.io/apiserver?

Reply:
Move it to ApplyTo